client/signup/steps/jpo-contact-form/index.jsx

```diff
@@ -0,0 +1,99 @@
+/**
+ * External dependencies
+ */
+import React, { PropTypes } from 'react';
+import { connect } from 'react-redux';
+import get from 'lodash/get';
+
+/**
+ * Internal dependencies
+ */
+import StepWrapper from 'signup/step-wrapper';
+import SignupActions from 'lib/signup/actions';
+import Card from 'components/card';
+import Button from 'components/button';
+import { translate } from 'i18n-calypso';
+
+import { setJPOContactForm } from 'state/signup/steps/jpo-contact-form/actions';
+
+import ContactUsGraphic from './contact-us-graphic';
+
+const JPOContactFormStep = React.createClass( {
+	propTypes: {
+		flowName: PropTypes.string,
+		goToNextStep: PropTypes.func.isRequired,
+		positionInFlow: PropTypes.number,
+		setJPOContactForm: PropTypes.func.isRequired,
+		signupProgress: PropTypes.array,
+		stepName: PropTypes.string,
+	},
+
+	onSelect() {
+		const jpoContactForm = true;
+
+		this.props.setJPOContactForm( jpoContactForm );
+
+		SignupActions.submitSignupStep( {
+			processingMessage: translate( 'Setting up your site' ),
+			stepName: this.props.stepName,
+			jpoContactForm
+		}, [], { jpoContactForm } );
+
+		this.props.goToNextStep();
+	},
+
+	skipStep() {
+		this.props.goToNextStep();
+	},
+
+	renderStepContent() {
+		return (
+			<div className="jpo__contact-form-wrapper">
+				<a className="jpo-homepage__select-news" href="#" onClick={ this.onSelect }>
+					<Card>
+						<ContactUsGraphic />
+						<Button onClick={ this.onSelect }>
+							{
+								translate( 'Add a contact form' )
```

Comment:
Hi! I've found a possible matching string that has already been translated **46** times:
`translate( 'Add Contact Form' )` *ES Score: 11*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).